fe/src/project/AspNetCore/areas/DownloadCenter/components/DownloadCenterCatalog/DownloadCenterCatalog.scss

```diff
@@ -0,0 +1,17 @@
+.dc-catalog {
+    &__container {
```

Comment:
**suggestion (non-blocking):** wouldn't it be better to give it a more meaningful name like &__category?